src/core/loggers/ObjectLogger.ts

```diff
@@ -31,7 +30,7 @@ export class ObjectLogger implements ObjectLoggerInterface {
     }
 
     public isOn(): boolean  {
-        if (MethodLogger.LOGGER_ON === false) {
+        if (ObjectLogger.LOGGER_ON === false) {
```

Comment:
how come this isnt "this."?
If its not inside an object isnt it possbile that this will act "singletonly", in other words in one part declares "on" an another "off" that only the last one will be true

Reply:
There is a static `logger switch`, and context `logger switch`.
I thought it like this, if you have more different loggers initialized, you can switch them one by one, but if you want to switch all of them, then you use static.